roboteam_ai/src/control/shotControllers/ShotController.cpp

```diff
@@ -107,46 +111,69 @@ bool ShotController::onLineToBall(const world::Robot &robot, const std::pair<Vec
 Vector2 ShotController::getPlaceBehindBall(const world::Robot &robot, const Vector2 &shotTarget) {
     Vector2 ballPos = world::world->getBall()->pos;
     Vector2 preferredShotVector = ballPos - shotTarget;
-    double distanceBehindBall = 2.0*Constants::ROBOT_RADIUS() + Constants::BALL_RADIUS();
+    double distanceBehindBall = 3.0 * Constants::ROBOT_RADIUS() + Constants::BALL_RADIUS();
```

Comment:
We might want to change this back, since they were quite slow with passing because of this.